app/templates/parachains.html

```diff
@@ -73,9 +72,21 @@ <h3>{{ network_name.capitalize() }} : {{ paras_count }} Paras: {{ parachain_coun
             "lengthMenu": [[30, -1], [30, "All"]]
         });
 
-        window.onboardParachain = (element, parachainId) => {
+        window.selectOrUnselectAllParachains = (element) => {
+            paraInputCheckboxes = document.querySelectorAll('input[name="para_id"]');
+            if (element.checked){
+                paraInputCheckboxes.forEach((paraInput) => paraInput.checked = true)
+            } else {
+                paraInputCheckboxes.forEach((paraInput) => paraInput.checked = false)
+            }
+        }
+
+        window.onboardSelectedParachains = (element) => {
+            paraInputCheckboxes = document.querySelectorAll('input[name="para_id"]');
+            selectedParaIds =  Array.from(paraInputCheckboxes).filter(input => input.checked).map(input => input.value);
```

Comment:
if parachain is already onboard will it onboard it again?

Reply:
Good point, we used to do this check in the front-end. Now this has to be implemented in the back-end.

Reply:
Implemented in https://github.com/paritytech/testnet-manager/pull/37/commits/646185821812cc45982e14fa8c39d25cb53d3dcb